README.md

```diff
@@ -37,7 +37,7 @@ To make Dropbox work in your self-hosted app, [go to this Wiki page](https://git
 
 The easiest way to clone all KeeWeb repos is:
 ```bash
-curl https://raw.githubusercontent.com/keeweb/keeweb/develop/dev-env.sh | bash -
+curl https://raw.githubusercontent.com/keeweb/keeweb/develop/dev-env.sh | SCHEMA=http bash -
```

Comment:
Let's leave the default option here.

Reply:
OK, done.. you know that _other_ user get ... _could not read_ .. ?
````
Cloning into 'keeweb'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
````

Reply:
Then we can make the https option default in the script, if we think it's what will work for the most of users.

Reply:
Should I patch? (I won't change the default behavior, because I don't know if this is used by a build bot)

Reply:
Sure! This script is only for humans who want to clone all the repos.